documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -342,15 +232,12 @@ Model In Image model files, application archives, and the WDT installation files
   
      If you don't see the `imagetool` directory under `/tmp/mii-sample/model-images` or the WDT installer is not in the ImageTool cache, then repeat the step to set up the WebLogic Image Tool in the [prerequisites]({{< relref "/samples/domains/model-in-image/prerequisites.md" >}}).
  
-     In a later step, we will specify corresponding domain resource attributes:
-     - `domain.spec.configuration.model.wdtInstallHome`
-       so that it references the WDT installation directory,
-       which the `imagetool.sh createAuxImage` command
-       defaults to `/auxiliary/weblogic-deploy`
-       (where `/auxiliary` is the default `--wdtHome` value for the `imagetool.sh createAuxImage` command).
-     - `domain.spec.configuration.model.modelHome`
-       so that it matches the `imagetool.sh createAuxImage` command's `--wdtModelHome` value,
-       which defaults to `/auxiliary/models`.
+     When you run this command, the image tool will create an auxiliary image with the specified model, variables, and archive files in the
+     image's `/auxiliary/models` directory. It will also add the latest version of the WDT installation in its `/auxiliary/weblogic-deploy` directory.
+     See [Create Auxiliary Image](https://oracle.github.io/weblogic-image-tool/userguide/tools/create-aux-image/) for additional image tool options.
+     The operator auxiliary image feature looks for WDT model and WDT install files in these specific directories by default; if you change
+     the location of these directories, then change the corresponding domain resource `spec.configuration.model.sourceModelHome` and
+     `spec.configuration.model.wdtModelHome` values.
```

Comment:
The domain resource values are incorrect.  Replace "corresponding domain resource..." with something like "corresponding domain resource auxiliary image [source location attributes](#source-locations)."